docs/releases/5.0.5.txt

```diff
@@ -30,3 +30,7 @@ Bugfixes
 
 * Fixed a bug in Django 5.0 that caused a migration crash when altering a
   ``GeneratedField`` referencing a rename field (:ticket:`35422`).
+
+* Fixed a bug in Django 5.0 that a misleading error was raised when the
+  required argument ``querysets`` of ``GenericPrefetch`` was not provided
+  (:ticket:`35426`).
```

Comment:
Previously if you did't pass in something for `querysets` you got a TypeError, and now you still get a TypeError (just a clearer one).
Therefore, I might have been too hasty in classifying this as a release blocker and should update this to be a cleanup ticket. 
If that's the case, I can remove the release note for 5.0.5 and instead have a release note for 5.1 about the signature change (but do not need to deprecate because not passing in  `querysets` was never really supported in the first place).

I would really love to hear @felixxm's opinion here 👍

Reply:
I would allow this as a bugfix in, for example, CPython :)
But, I don't know much about Django development process.

Both sound fine to me, please guide me! 😊

Reply:
> Previously if you did't pass in something for `querysets` you got a TypeError, and now you still get a TypeError (just a clearer one). Therefore, I might have been too hasty in classifying this as a release blocker and should update this to be a cleanup ticket. If that's the case, I can remove the release note for 5.0.5 and instead have a release note for 5.1 about the signature change (but do not need to deprecate because not passing in `querysets` was never really supported in the first place).
> 
> I would really love to hear @felixxm's opinion here 👍

IMO it's a release blocker. `GenericPrefetch` should not crash when acceptable arguments are passed.

Reply:
Thank you, makes sense 👍